src/vm/ClrEtwAll.man

```diff
@@ -1705,6 +1707,42 @@
                         </UserData>
                     </template>
 
+                    <template tid="AssemblyLoadContextResolvingHandlerInvoked">
```

Comment:
FYI it'll be nice to follow-up with a PR to PerfView with corresponding changes to TraceEvent for these new events (specifically `ClrTraceEventParser.cs`)

Reply:
Yep. Planning on doing that once we have all the binder tracing events in (few more to go).